tasks/system/Linux.yml

```diff
@@ -83,9 +83,16 @@
     systemd:
       daemon_reload: True
 
-  - name: "Start SonarQube server"
+  - name: "Enable sonarqube.service"
     service:
       name: "sonarqube"
       enabled: True
 
+  - name: "Render nginx config"
```

Comment:
split http/https servers and define in name that it's nginx configs

Reply:
Done